hermes/probe/monitor.go

```diff
@@ -110,7 +108,7 @@ func (mp *MonitorProbe) Start(ctx context.Context, metricChan chan<- *cpmetrics.
 	}
 }
 
-// reportMetrics sends the metrics recorded in the current probe run to Cloudprober.
+// Sends the metrics recorded in the current probe run to Cloudprober.
```

Comment:
GoDoc expects the first word of the comment to match the identifier it belongs to.
Here and elsewhere.

Reply:
Ack.